### DISCUSSION TITLE
Lora gallery not displaying any Loras

### DISCUSSION THREAD

**Discussion by popsoda27** [lshqqytiger/stable-diffusion-webui-directml#511] [Q&A]
I have a weird issue with Pony Loras not appearing in the Lora UI tab.
The Checkpoint and TI tabs have no issue picking up the respective Pony files but Pony Loras are not showing at all.

While I'm was on SD 1.5, usually when I download a new checkpoint or TI or Lora, I use Civitai Helper extension to scan them in, do a refresh and they immediately appear in their respective tabs. Recently I moved on to Pony. Same thing, download and place the files the right folders, scan with Civitai Helper, the checkpoints and TIs appear, only Loras are still missing in its tab. Anyone know what's wrong?

**Comment by VeteranXT** [comment]
There is lora that is not compatible with models. Not all loras are. 
Pony lora only works on Pony models.
